src/main/java/org/gridsuite/shortcircuit/server/repositories/FaultResultRepository.java

```diff
@@ -28,4 +26,17 @@ public interface FaultResultRepository extends JpaRepository<FaultResultEntity,
 
     @EntityGraph(attributePaths = {"feederResults"}, type = EntityGraphType.LOAD)
     Set<FaultResultEntity> findAllWithFeederResultsByFaultResultUuidIn(List<UUID> faultResultsUUID);
+
+    @Modifying
+    @Query(value = "DELETE FROM fault_result_entity WHERE result_result_uuid = ?1", nativeQuery = true)
```

Comment:
This request takes about 400ms for 7000 fault results to delete. 
While deleting feeder results lasts only 10ms over 25 000 results